Events/EddiEvents.csproj

```diff
@@ -197,6 +197,7 @@
     <Compile Include="DockedEvent.cs" />
     <Compile Include="JumpedEvent.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="FSSAllBodiesFound.cs" />
```

Comment:
Our convention is to name events to match the name they'll have in the app. In this case, we should probably use the name `SystemScanComplete`.

Reply:
Requested change completed.